k8/configs/configmap-genai.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: genai-config
+  namespace: team-promptheus
+data:
+  MEILISEARCH_URL: "http://meilisearch-service:7700"
```

Comment:
things like connection relevant environment variables are better put one level up given that this way it is more apparent how the system is wired.

Using ConfigMaps for settings is fine though.